recipe for target 'menu' failed

Hi,

I have setup a vagrant dev enviroment and that works fine initially. If I do a 'vagrant halt' and 'vagrant up' and login to the box I am unable to start ralph3.

The error I am getting is this:

```
(venv) vagrant@ralph-dev:~/ralph$ make menu
ralph sitetree_resync_apps
Traceback (most recent call last):
  File "/home/vagrant/venv/bin/ralph", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/home/vagrant/venv/lib/python3.5/site-packages/pkg_resources/__init__.py", line 3088, in <module>
    @_call_aside
  File "/home/vagrant/venv/lib/python3.5/site-packages/pkg_resources/__init__.py", line 3072, in _call_aside
    f(*args, **kwargs)
  File "/home/vagrant/venv/lib/python3.5/site-packages/pkg_resources/__init__.py", line 3101, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/vagrant/venv/lib/python3.5/site-packages/pkg_resources/__init__.py", line 574, in _build_master
    ws.require(__requires__)
  File "/home/vagrant/venv/lib/python3.5/site-packages/pkg_resources/__init__.py", line 892, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/vagrant/venv/lib/python3.5/site-packages/pkg_resources/__init__.py", line 778, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'ralph' distribution was not found and is required by the application
Makefile:58: recipe for target 'menu' failed
make: *** [menu] Error 1
```


I am getting the same error when I run 'make run' I am unsure how to continue.

--- a/vagrant/Vagrantfile
+++ b/vagrant/Vagrantfile
@@ -10,8 +10,20 @@ Vagrant.configure(2) do |config|
     config.vm.box = "ubuntu/xenial64"
     config.vm.hostname = "ralph-dev"
     config.vm.synced_folder "../", ralph_dir, type: "rsync",
-        rsync__exclude: ["../node_modules", "../.git", "../bower_components"],
-        rsync__auto: true
+        rsync__auto: true,
+        rsync__exclude: [
+            ".git",
+            "node_modules",
+            "bower_components",
+            "src/ralph.egg-info",
+            "src/ralph/settings/local.py", 
+            "src/ralph/static/css",
+            "src/ralph/static/vendor",
+            "src/ralph/admin/static/bower_components",
+            "src/ralph/admin/static/elements/elements-min.html",
+            "*__pycache__",
+            "*.pyc"
+            ]
     config.vm.provision :shell, path: "configure_kernel.sh", privileged: true
     config.vm.provision :reload
     config.vm.provision :shell, path: "provision.sh", privileged: false,

